code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -286,3 +287,43 @@
 	. = ..()
 	pizza = new /obj/item/reagent_containers/food/snacks/pizza/meat(src)
 	boxtag = "Meatlover's Supreme"
+
+//An anomalous pizza box that, when opened, produces the opener's favorite kind of pizza.
+/obj/item/pizzabox/infinite
+	resistance_flags = FIRE_PROOF | LAVA_PROOF | ACID_PROOF //hard to destroy

```

Comment:
Why does this need all these?

Reply:
it's anomalous

Reply:
"There was a big plasma fire that destroyed the department and all of our pizza boxes. Except one. We opened it up and there was a fresh pizza inside... can we keep it?!"

Reply:
So you eat all the pizza